src/sentry/api/helpers/group_index/update.py

```diff
@@ -633,6 +336,314 @@ def update_groups(
     return Response(result)
 
 
+def handle_resolve_in_release(
+    status: str,
+    status_details: Mapping[str, Any],
+    group_list: Sequence[Group],
+    projects: Sequence[Project],
+    project_lookup: Mapping[int, Project],
+    acting_user: User | None,
+    user: User | RpcUser,
+    result: MutableMapping[str, Any],
+) -> tuple[dict[str, Any], GroupResolution.Type | None] | Response:
```

Comment:
Notice that we either return data or a response. This is handled at the caller level:
<img width="314" alt="image" src="https://github.com/user-attachments/assets/98c89889-3ad6-4291-bf75-bfdcdf90bdaa">